node.gyp

```diff
@@ -616,13 +608,9 @@
       'target_name': 'cctest',
       'type': 'executable',
       'dependencies': [ 'deps/gtest/gtest.gyp:gtest' ],
-      'conditions': [
-        [ 'node_shared_v8=="false"', {
-          'dependencies': [ 'deps/v8/tools/gyp/v8.gyp:v8' ],
```

Comment:
I added it next to the cares dep further up in the patch. Will take a second look.